tools/d2g/README.md

```diff
@@ -1,15 +1,14 @@
 # d2g
 
-The `d2g` repo is intended to contain the following tooling:
-
-  * A go library for converting Docker Worker task payloads into Generic Worker task payloads
-  * A CLI wrapper for the above go library
-
-Eventually this library will be used in Generic Worker, in order to support Docker Worker
-task payloads. On detecting that a claimed task has a Docker Worker payload, it will use
-the d2g library to convert the payload into its own native format, before validating and executing it.
-
-The intention is that this library will also be helpful to users wishing to
-convert Docker Worker task payloads directly to Generic Worker native task
-payloads, as it is intended to eventually deprecate the Docker Worker payload
-format entirely.
+d2g is a go library for converting Docker Worker task payloads into Generic
```

Comment:
It can also do full task definitions. Probably worth adding to this readme somewhere.